client/src/components/Supporters.js

```diff
@@ -0,0 +1,76 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Button, ProgressBar } from '@freecodecamp/react-bootstrap';
+
+import { commaNumber } from '../utils';
+import FullWidthRow from '../components/helpers/FullWidthRow';
+import Spacer from '../components/helpers/Spacer';
+
+import './supporters.css';
+
+const propTypes = {
+  activeDonations: PropTypes.number.isRequired,
+  isDonating: PropTypes.bool.isRequired
+};
+
+function Supporters({ isDonating, activeDonations }) {
+  return (
+    <Fragment>
+      <FullWidthRow>
+        <h2>Support an open future.</h2>
+      </FullWidthRow>
+      <FullWidthRow>
+        <div id='supporter-progress-wrapper'>
+          <ProgressBar max={10000} now={activeDonations} />
```

Comment:
What happens if `activeDonations` becomes greater `max`?

Reply:
Nice catch.

@QuincyLarson How would we want to handle something like

![image](https://user-images.githubusercontent.com/18572518/49367805-fd73c800-f6e4-11e8-8b87-1ddb6ff9fc9f.png)

Reply:
When we cross the goal: 

- Remove the progress bar.
- I think we can add a `🎉` and thank you message with the count, in a larger font.

Anyways that's all a good to have?

Reply:
Like this?

![image](https://user-images.githubusercontent.com/18572518/49371234-b7bbfd00-f6ee-11e8-828e-561db7765fb3.png)
![image](https://user-images.githubusercontent.com/18572518/49371274-d7ebbc00-f6ee-11e8-815f-cbc3943cf538.png)

Reply:
LGTM